### PR TITLE
Adding multipart/form-data support

### PR DESCRIPTION
**Pull request by Orbifoldt**
Adding support for code generation of models for specs that contain multipart/form-data requests:
- Added qute template for multipartbody pojo's
- Changed api.qute to support this pojo
- Added a skipFormModel parameter to the configuration to allow generation of models for form-params

I followed the approach from [Quarkus - Using the REST Client with Multipart](https://quarkus.io/guides/rest-client-multipart) and [RESTEasy JAX-RS specifications](https://docs.jboss.org/resteasy/docs/4.7.5.Final/userguide/html_single/index.html#multipartform_annotation) as this seems the most straight forward. 

Let me know if you see any cleaner ways to accomplish any of these features :)

### CODE REVIEW

File: deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapperTest.java [hunk 1]
```diff
@@ -114,15 +99,72 @@ void circuitBreaker() throws URISyntaxException, FileNotFoundException {
...
-
-        OpenApiClientGeneratorWrapper generatorWrapper = new OpenApiClientGeneratorWrapper(simpleOpenApiFile, targetPath)
+        OpenApiClientGeneratorWrapper generatorWrapper = createGeneratorWrapper("simple-openapi.json")
                 .withCircuitBreakerConfiguration(Map.of(
                         "org.openapitools.client.api.DefaultApi", List.of("opThatDoesNotExist", "byeGet")));
 
         return generatorWrapper.generate();
     }
+
+    private OpenApiClientGeneratorWrapper createGeneratorWrapper(String specFileName) throws URISyntaxException {
```
Comment (ricardozanini) [inline deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapperTest.java:109]: I'm currently working on an issue that I did the same refactoring. Many thanks! Let's keep yours, I'll rebase after merging this PR.

**Review by ricardozanini — APPROVED** [review]
Looks great! Many thanks!

**Comment by ricardozanini** [comment]
@all-contributors please add @Orbifoldt for code

**Review by ricardozanini — COMMENTED** [review]
One more thing: I think we can get rid of the [formsParam.qute](https://github.com/quarkiverse/quarkus-openapi-generator/blob/main/deployment/src/main/resources/templates/formParams.qute) template since it won't be used anymore.

**Comment by Orbifoldt** [comment]
@ricardozanini Removed the formParams.qute, good catch!

**Comment by ricardozanini** [comment]
@Orbifoldt many thanks! Please let me know if you need a patch release for this feature.

**Pull request merged**